content/en/logs/log_collection/csharp.md

```diff
@@ -293,20 +299,141 @@ logs:
+## Agentless logging with APM
 
-It is possible to stream logs from your application to Datadog or to the Datadog Agent directly. This is not the recommended setup as handling connection issues should not be done directly in your application, but it might not be possible to log to a file when your application is running on a machine that cannot be accessed.
-{{< tabs >}}
-{{% tab "Serilog" %}}
+It is possible to stream logs from your application to Datadog directly, without making any code changes, using the .NET APM automatic instrumentation library. This approach sends logs directly to Datadog, so it does not benefit from [features such as sensitive data scrubbing][3] which are provided by the Datadog Agent. For that reason, we recommend using file tail logging where possible, but it is useful in environments where this is not possible (when using [Azure App Service][4] for example).
+
+Agentless logging (also known as "direct log submission") supports the following frameworks:
+- Serilog (v1.0+)
+- NLog (v2.1+)
+- log4net (v1.0+)
+- Microsoft.Extensions.Logging (2.0+)
+
+It does not require modifying your application code, or installing additional dependencies into your application.
+
+### Configure the APM library
+
+Agentless logging is only available when using APM with automatic instrumentation. To get started, instrument your application as described in the following documents
+
+- [.NET Core/.NET 5+ applications][5]
+- [.NET Framework applications][6]
+
+After installing, verify you are receiving traces correctly.
+
+### Enable agentless logging
+
+To enable agentless logging, set the following environment variables:
+
+`DD_API_KEY`
+: Your [Datadog API Key][7] for sending your logs to Datadog.
+
+`DD_SITE`
+: The name of [your Datadog site][8]. Choose from one of the following examples:<br>
+**Example**: `datadoghq.com` (US1), `datadoghq.eu` (EU), `us3.datadoghq.com` (US3), `us5.datadoghq.com` (US5), `ddog-gov.com` (US1-FED) <br>
+**Default**: `datadoghq.com` (US1)
+
+`DD_LOGS_INJECTION`
+: Enables [connecting logs and traces][2]:<br>
+**Default**: `true` <br>
+Enabled by default when using agentless logging from Tracer version 2.7.0.
+
+`DD_LOGS_DIRECT_SUBMISSION_INTEGRATIONS`
+: Enables agentless logging. Enable for your logging framework by setting to `Serilog`, `NLog`, `Log4Net`, or `ILogger` (for `Microsoft.Extensions.Logging`). If you are using multiple logging frameworks, use a semicolon separated list of variables.<br>
+**Example**: `Serilog;Log4Net;NLog`
 
-Install the Datadog [Serilog sink][1], which sends events and logs to Datadog. By default the sink forwards logs through HTTPS on port 443.
+<div class="alert alert-warning">
+  <strong>Note:</strong> If you are using a logging framework in conjunction with `Microsoft.Extensions.Logging`, you will generally need to use the framework name. For example, if you are using [Serilog.Extensions.Logging][8], you should set `DD_LOGS_DIRECT_SUBMISSION_INTEGRATIONS=Serilog`.
```

Comment:
The `[Serilog.Extensions.Logging][8]` didn't get picked up correctly since this is inside a custom HTML block. You'll need to write the anchor tag manually

Reply:
Thanks, fixed in https://github.com/DataDog/documentation/pull/13907/commits/1a70bc07eeaf48a2d0e0d514ac2e270405e25364. 👍